pom.xml

```diff
@@ -81,6 +81,7 @@
         <module>presto-redis</module>
         <module>presto-cassandra</module>
         <module>presto-blackhole</module>
+        <module>presto-in-memory</module>
```

Comment:
On the other hand there is `presto-local-file`.  `presto-in-memory` looks better and is more readable (at least for me). Same applies to for example properties: `inmemory.splits-per-node` is worse compared to `in-memory.splits-per-node`.